mango/container/mp.py

```diff
@@ -444,6 +449,21 @@ def _find_sp_queue(self, aid):
         raise ValueError(f"The aid '{aid}' does not exist in any subprocess.")
 
     def create_agent_process(self, agent_creator, container, mirror_container_creator):
```

Comment:
Regarding the design, it makes sense that delayed starting agent processes is an additional feature. I do not feel the prior approach was bad. Being able to start the process at will before starting the container makes sense as it is closer to the normal agent lifecycle. So I would suggest to introduce a new method for delayed agent process start (or a flag?). This also would prevent breaking the API.